data/vibe/data/bson.d

```diff
@@ -1229,7 +1229,7 @@ unittest {
 @safe unittest {
 	static struct A { int value; static A fromJson(Json val) @safe { return A(val.get!int); } Json toJson() const @safe { return Json(value); } Bson toBson() { return Bson(); } }
 	static assert(!isStringSerializable!A && isJsonSerializable!A && !isBsonSerializable!A);
-	static assert(!isStringSerializable!(const(A)) && !isJsonSerializable!(const(A)) && !isBsonSerializable!(const(A)));
+	static assert(!isStringSerializable!(const(A)) && isJsonSerializable!(const(A)) && !isBsonSerializable!(const(A)));
```

Comment:
This test changed as it can be serialized to Json using const struct just fine.

Reply:
The thing here is that the `isXSerializable` traits work bi-directionally (i.e. both, serialization and deserialization must be possible) and . I'd have to think about possibilities how this change might or might not break existing code, but my initial guess would be that this should be fine in all typical cases.

I'd just apply the `Unqual` change to `isStringSerializable` and `isBsonSerializable` as well to keep the behavior consistent.

Reply:
changed with some more unittest changes